script/main.js

```diff
@@ -393,6 +393,7 @@ function ResetTracker() {
     chests.forEach(chest => delete chest.isOpened);
     dungeons.forEach(dungeon => Object.values(dungeon.chestlist).forEach(chest => delete chest.isOpened));
     items = Object.assign(baseItems);
+    medallions = defaultMedallions;
```

Comment:
This is the only logical change.  All other changes were my editor automatically cleaning up trailing whitespace

Reply:
Closing because this is should be a deep copy instead of a shallow one